packages/reports/addon/components/common-actions/schedule.hbs

```diff
@@ -59,21 +59,56 @@
           Close
         </DenaliButton>
       {{else}}
-        <DenaliInputGroup
-          @label="Recipients"
-          @isStacked={{true}}
-          class="{{unless this.isRuleValid "schedule__modal-recipients--invalid"}} m-b-30"
-        >
-          <NaviEmailInput
-            class="schedule__modal-input--recipients w-full"
-            @emails={{readonly this.localDeliveryRule.recipients}}
-            @onUpdateEmails={{this.updateRecipients}}
-            @isDisabled={{is-rejected this.deliveryRule}}
-          />
-          <div class="schedule__modal-helper-recipients">
-            Please enter valid email addresses separated by commas.
+        <DenaliInputGroup @label="Delivery" class="m-b-30">
+          <PowerSelect
+            class="w-full"
+            @triggerClass="schedule__modal-delivery-trigger w-full"
+            @dropdownClass="w-full"
+            @options={{this.deliveryOptions}}
+            @selected={{this.localDeliveryRule.delivery}}
+            @renderInPlace={{true}}
+            @onChange={{fn (mut this.localDeliveryRule.delivery)}}
+            as | delivery |
+          >
+            {{capitalize delivery}}
+          </PowerSelect>
+          <div class="schedule__modal-helper-delivery">
+              If 'Email' is selected emails will be automatically delivered to the designated recipients, to not recieve emails select 'None'.
```

Comment:
Can we show only the description of the option selected?

Reply:
Email - `Send this ${report/dashboard} as email on a recurring basis`
None - `Schedule and cache this ${report/dashboard} without delivery`